### PR TITLE
Ticket #6129: Change MediaUtils to take data directly from Pender

### PR DESCRIPTION
**Pull request by danielafeitosa**

### CODE REVIEW

File: src/app/components/media/MediaUtil.js [hunk 1]
```diff
@@ -77,30 +77,11 @@ const MediaUtil = {
...
-      return data.user ? data.user.name : '';
-    case 'instagram.com':
-      return data.author_name;
-    case 'facebook.com':
-      return data.user_name;
-    default:
-      return data.username || media.domain;
-    }
+    return data.author_name || media.domain;
   },
```
Comment (caiosba) [inline src/app/components/media/MediaUtil.js]: Did you verify that the _behavior_ is still the same?

File: src/app/components/media/MediaUtil.js [hunk 2]
```diff
@@ -77,30 +77,11 @@ const MediaUtil = {
...
-      return `@${data.username}`;
-    case 'facebook.com':
-      return data.username;
-    case 'youtube.com':
-      return '';
-    default:
-      return data.username;
-    }
+    return data.username;
   },
```
Comment (caiosba) [inline src/app/components/media/MediaUtil.js]: Did you verify that the _behavior_ is still the same?

File: src/app/components/media/MediaUtil.js [hunk 3]
```diff
@@ -221,20 +202,7 @@ const MediaUtil = {
...
-      case 'instagram.com':
-        return data.picture;
-      case 'youtube.com':
-        return data.picture;
-      }
-    } catch (e) {
-      return null;
-    }
+    return data.picture;
   },
```
Comment (caiosba) [inline src/app/components/media/MediaUtil.js]: Did you verify that the _behavior_ is still the same?

File: src/app/components/media/MediaUtil.js (hunk 1 above)
Comment (danielafeitosa) [inline src/app/components/media/MediaUtil.js]: yes :)

File: src/app/components/media/MediaUtil.js (hunk 2 above)
Comment (danielafeitosa) [inline src/app/components/media/MediaUtil.js]: yes :)

File: src/app/components/media/MediaUtil.js (hunk 3 above)
Comment (danielafeitosa) [inline src/app/components/media/MediaUtil.js]: yes :)

**Pull request merged**